Casamari/MNC007.xml

```diff
@@ -36,76 +39,168 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                           reason="lost" resp="SH">ል</supplied>ደ፡ ቂር<gap reason="illegible"
+                           unit="chars" quantity="2"/></explicit>
+                     <note>Most probably the name of <persName ref="PRS12029GabraMa">Gabra
+                           Māryām</persName> is stated here at the very end together with that of a scribe,
+                      probably named Walda Qirqos (<ref target="#h3"/>).</note>
                   </msItem>
-                  <msItem xml:id="ms_i3">
-                     <locus from="21v" to="32r"/>
-                     <title type="complete">Malkǝʾa Māryām</title>
+
+                  <msItem xml:id="ms_i4" corresp="#h2">
+                     <locus from="23r" to="32r"/>
+                     <title type="complete" ref="LIT2891RepCh172"><foreign xml:lang="gez">Malkǝʾa
+                           Māryām</foreign></title>
                      <textLang mainLang="gez"/>
-                     <note>
-                     </note>                
+                     <note>The work comprises 60 stanzas.</note>
+                     <incipit xml:lang="gez">
+                        <locus target="#23r"/>
+                        <pb n="23r"/>
+                        <hi rend="rubric">ሰላም፡ ሰላም፡</hi> ለዝክረ፡ ስምኪ፡ ሐዋዝ፡ እምነ፡ ከልበኔ፡ ወቍስጥ፡ ወእምነ፡
+                        ሰንበልት፡ ምዑዝ፡ <hi rend="rubric">ማርያም፡</hi> ለባሲተ፡ ዓቢይ፡ ትእዛዝ፡ ይስቅየኒ፡ ለለጽባሁ፡ ወይነ፡
+                        ፍቅርኪ፡ አዚዝ፡ ከመ፡ ይሰቅዮ፡ ውሂዝ፡ ዘሰናይ፡ አርዝ ።</incipit>
+                     <explicit xml:lang="gez">
+                        <locus target="#32r"/>
+                        <hi rend="rubric">ስብሐት፡</hi> ለኪ፡ <hi rend="rubric">ማርያም፡</hi> በኍልቈ፡ ኵሉ፡
+                        ስእርትየ፡ <hi rend="rubric">ስብሐት፡</hi> ለኪ፡ <hi rend="rubric">ማርያም፡</hi> በኍልቈ፡
+                        ኵሉ፡ አእጽምትየ፡ <hi rend="rubric">ስብሐት፡</hi> ለኪ፡ <hi rend="rubric">ማርያም፡</hi>
+                        በኍልቈ፡ ኵሉ፡ ዘተርእየ፡ ስብሐት፡ ለኪ፡ ንግሥትየ፡ በኍልቈ፡ ኵሉ፡ ዘኢያስተርአየ፡ ስብሐተ፡ ድንግልናኪ፡ ዘልፈ፡
+                        ይነግር፡ አፉየ።</explicit>
                   </msItem>
-                  <msItem xml:id="ms_i4">
+
+                  <msItem xml:id="ms_i5" corresp="#h2">
                      <locus from="32v" to="41v"/>
-                     <title type="complete" ref="LIT2839RepCh123">Malkǝʾa ʾIyasus</title>
+                     <title type="complete" ref="LIT2839RepCh123"><foreign xml:lang="gez">Malkǝʾa
+                           ʾIyasus</foreign></title>
                      <textLang mainLang="gez"/>
-                     <note>
-                        <persName ref="PRS12028WaldaAr">Walda ʾAragāwi</persName> is mentioned in the supplication formula at the end of the text.
-                     </note>             
+                     <note>The work comprises 55 stanzas.</note>
+                     <incipit xml:lang="gez">
+                        <locus target="#32v"/>
+                        <pb n="32v"/>
```

Comment:
Actually, i don't know. I learned this from Dorothea. It states that the text start with the first line on this leaf.

Reply:
Ops, I have never been using this at the beginning of the folio. I thought it was taken from granted or optional.

Reply:
I would not use it.

Reply:
We can keep this practice as optional in XML records, provided that we establish that it will not be visualized in the printed catalogue (in my view it might look redundant to the reader as it would follow the identical information: incipit, f. 32v)